### ISSUE TITLE
citation

### ISSUE THREAD

**Issue by camiloariza** [luferupi8811/ESP_AdaptiveController#1]
The citation section in the README lists two BibTeX entries. I'm not able to find a direct relation of this work with the `Chaudhry2013` citation. Must the two citation entries be included when referencing your work?

Thank you in advance.

**Comment by luferupi8811** [comment]
> The citation section in the README lists two BibTeX entries. I'm not able to find a direct relation of this work with the `Chaudhry2013` citation. Must the two citation entries be included when referencing your work?
> 
> Thank you in advance.

Dear camiloariza,

Thanks for your question. In the file train_controller.m, I used the backpropagation and forward-propagation codes that were originally written by Chaudhry2013 to perform the offline training of an ANN. In this work, I adapted those codes to perform the online training of an ANN working as an adaptive controller. I consider that the work of Chaudhry2013 was useful, and that is why I included it in the list of BibTeX entries.
However, I understand your doubt so that I will remove this last work from the list, and I will recommend its citation in the case that only the backpropagation and forward-propagation codes are used.

Thanks a lot.
